### PR TITLE
Fix linux documentation and config names

### PR DESCRIPTION
**Pull request by fearful-symmetry**
## What does this PR do?

- This fixes a handful of documentation issues in the linux module relating to config labels and the readme.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [X] I have verified that all datasets collect metrics or logs.

## Related issues

- https://github.com/elastic/integrations/issues/546
- https://github.com/elastic/integrations/issues/547
- https://github.com/elastic/integrations/issues/548

### CODE REVIEW

File: packages/linux/_dev/build/docs/README.md [hunk 1]
```diff
@@ -11,6 +11,38 @@ permissions can be blocked by
...
+
+### KSM
+
+The KSM module reports data from https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html[Kernel Samepage Merging]. 
+In order to take advantage of KSM, applications must use the `madvise` system call to mark memory regions for merging. KSM is not enabled on all distros, and KSM status is set with the `CONFIG_KSM` kernel flag.
+
+
+### Memory
+
+The memory metricset extends system/memory and adds linux-specific memory metrics, including Huge Pages and overall paging statistics.
```
Comment (mtojek) [inline packages/linux/_dev/build/docs/README.md:31]: I think you should replace all references to metricsets (and potentially filesets) with data streams.

File: packages/linux/_dev/build/docs/README.md [hunk 2]
```diff
@@ -11,6 +11,38 @@ permissions can be blocked by
...
+
+### Pageinfo
+
+The pageinfo metricset reports on paging statistics as found in /proc/pagetypeinfo
```
Comment (mtojek) [inline packages/linux/_dev/build/docs/README.md:35]: `/proc/pagetypeinfo` ? 

nit: "." at the end of sentence

File: packages/linux/_dev/build/docs/README.md [hunk 3]
```diff
@@ -50,7 +82,7 @@ This dataset is available on:
 
 This dataset requires kernel 2.6.14 or newer.
```
Comment (mtojek) [inline packages/linux/_dev/build/docs/README.md:83]: Yeah, datasets became data streams :)

File: packages/linux/_dev/build/docs/README.md [hunk 4]
```diff
@@ -11,6 +11,38 @@ permissions can be blocked by
...
+The conntrack module reports on performance counters for the linux connection tracking component of netfilter. 
+Conntrack uses a http://people.netfilter.org/pablo/docs/login.pdf[hash table] to track the state of network connections.
+
+### Iostat
+
+The iostat module reports per-disk IO statistics that emulate `iostat -x` on linux.
+
+### KSM
+
+The KSM module reports data from https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html[Kernel Samepage Merging].
```
Comment (mtojek) [inline packages/linux/_dev/build/docs/README.md:25]: These hyperlinks need some adjustments. Please take a look at the Apache integration:

```
[Apache](https://httpd.apache.org/)
```

Not sure how will Kibana render ones like `https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html[Kernel Samepage Merging`

File: packages/linux/_dev/build/docs/README.md [hunk 5]
```diff
@@ -11,6 +11,38 @@ permissions can be blocked by
 
 ## Metrics
 
+### Conntrack
+
+The conntrack module reports on performance counters for the linux connection tracking component of netfilter.
```
Comment (mtojek) [inline packages/linux/_dev/build/docs/README.md:16]: nit: Linux or linux? (consistency across the document)

**Review by mtojek — COMMENTED** [review]
Only minor issues.

**Review by andrewkroh — APPROVED** [review]
LGTM

**Pull request merged**